website/docs/r/content_library_item.html.markdown

```diff
@@ -17,22 +17,36 @@ data "vcfa_content_library" "cl" {
   name = "My Library"
 }
 
-resource "vcfa_content_library_item" "cli" {
-  name               = "My OVA"
-  description        = "My uploaded OVA"
-  content_library_id = data.vcfa_content_library.cl.id
-  upload_piece_size  = 10
-  file_path          = "./photon.ova"
+resource "vcfa_content_library_item" "ova" {
+  name               = "my-ova"
+  description        = "Description of my-ova"
+  content_library_id = vcfa_content_library.cl.id
+  files_paths        = ["./my_ova.ova"]
 }
+
+resource "vcfa_content_library_item" "iso" {
+  name               = "iso1"
+  description        = "Description of iso1"
+  content_library_id = vcfa_content_library.cl.id
+  files_paths        = ["./linux.iso"]
+}
+
+resource "vcfa_content_library_item" "ovf" {
+  name               = "ovf"
+  description        = "Description of OVF"
+  content_library_id = vcfa_content_library.cl.id
+  files_paths        = ["./my-ovf/descriptor.ovf", "./my-ovf/disk1.vmdk"]
+}
+
 ```
 
 ## Argument Reference
 
 The following arguments are supported:
 
 * `name` - (Required) The name of the Content Library Item
-* `content_library_id` - (Required) ID of the Content Library that this Content Library Item belongs to
-* `file_path` - (Required) Path to the OVA/ISO to create the Content Library Item
+* `content_library_id` - (Required) ID of the [Content Library](/providers/vmware/vcfa/latest/docs/resources/content_library) that this Content Library Item belongs to
+* `files_paths` - (Required) A single path to an OVA/ISO, or multiple paths for an OVF and its referenced files, to create the Content Library Item
```

Comment:
Somehow missed this single letter. It should really be called file_paths. In English when a noun modifies another noun, the first one usually remains singular.